dags/datasets/datastore-resource-description.py

```diff
@@ -0,0 +1,72 @@
+
+from datetime import datetime, timedelta
+from textwrap import dedent
+import requests
+import logging
+
+# The DAG object; we'll need this to instantiate a DAG
+from airflow import DAG
+from airflow.models import Variable
+from airflow.operators.python import PythonOperator
+from ckan_operators.package_operator import GetCkanPackageListOperator, CheckCkanResourceDescriptionOperator, PrintCkanResponseOperator, GetAllPackagesOperator
+from utils_operators.slack_operators import GenericSlackOperator
+
+# init hardcoded vars for these dags
+ACTIVE_ENV = Variable.get("active_env")
+CKAN_CREDS = Variable.get("ckan_credentials_secret", deserialize_json=True)
+CKAN = CKAN_CREDS[ACTIVE_ENV]["address"]
+CKAN_APIKEY = CKAN_CREDS[ACTIVE_ENV]["apikey"]
+
+# Operators; we need this to operate!
+from airflow.operators.bash import BashOperator
+with DAG(
+    'description',
+    # These args will get passed on to each operator
+    # You can override them on a per-task basis during operator initialization
+    default_args={
```

Comment:
I didnt teach you about DAG default_args 😅 They let you define some basic things about the DAG overall

Look at some existing default_args in other DAGs and update this. I recommend looking here for inspiration:
https://github.com/open-data-toronto/operations/blob/b918d84cf9a3c694b8155949b691761ca0cba01f/dags/datasets/files_to_datastore/files_to_datastore.py#L382

Reply:
Will take a look and update👍